src/apb/engine.py

```diff
@@ -473,20 +473,12 @@ def cmdrun_init(**kwargs):
     else:
         app_name = apb_name
 
-    organization = kwargs['org']
-    if organization is None:
-        if len(apb_tag_arr) > 1:
```

Comment:
Since we no longer need `apb_tag_arr` I suggest on lines 469-470 we instead have:
apb_name = kwargs['tag'].